api/controllers/UserController.js

```diff
@@ -120,5 +120,21 @@ module.exports = _.mapValues({
     const params = req.allParams();
     Validation.requireParams(params, 'name');
     return res.ok(await Validation.usernameAvailable(params.name));
-  }
+  },
+  async forgotUsername (req, res) {
+    const params = req.allParams();
+    Validation.requireParams(params, 'email');
+    const user = await User.findOne({email: params.email});
+    if (!user) return res.notFound();
```

Comment:
Hmmmm...yes, good point, but we already do this for the password check. So that should be removed.